.github/workflows/test_k6.yml

```diff
@@ -0,0 +1,66 @@
+# See: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+
+name: 'Test: k6'
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - '.github/workflows/test_k6.yml'
+      - 'k6/**'
+      - 'services/.env.example'
+      - 'services/**'
+      - '!services/**.test.ts'
+      - '!services/**.spec.ts'
+      - '!**.md'
+
+env:
+  k6TestsPath: 'k6/'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js version
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: '${{ env.k6TestsPath }}/.node-version'
+          cache: 'npm'
+          cache-dependency-path: '${{ env.k6TestsPath }}/package-lock.json'
+
+      - name: Set ENV-variables for test-environment
+        run: |
+          cp ./services/.env.example services/.env
+
+      - name: Run Services with Docker
+        working-directory: ./services
+        run: |
+          docker --log-level 'warn' compose -f docker-compose.yml up -d --quiet-pull --wait --wait-timeout 300
+
+      - name: Install k6 dependencies
+        working-directory: ${{ env.k6TestsPath }}
+        run: |
+          npm ci --omit=optional --no-fund --no-audit
+
+      - name: Lint k6 tests
+        working-directory: ${{ env.k6TestsPath }}
+        run: 'npm run lint'
+
+      - name: Download k6
+        working-directory: ${{ env.k6TestsPath }}
+        run: |
+          curl https://github.com/grafana/k6/releases/download/v0.51.0/k6-v0.51.0-linux-amd64.tar.gz -L | tar xvz --strip-components 1
+
+      - name: Run k6 tests
+        working-directory: ${{ env.k6TestsPath }}
+        run: |
+          npx dotenv -e ../services/.env -- ./k6 run tests/pvProgramPerformance.js
+          npx dotenv -e ../services/.env -- ./k6 run tests/heapMemory.js
+
```

Comment:
It's on purpose we don't test k6/tests/payment100kRegistrationOCW.js here right?

Reply:
Yes indeed the 100k payment and registration test takes locally around 7 minutes, and this tests represent two issues we talked about couple of months ago. It is a look into performance metrics. We could change them tho to anything else. It can be any scenario.

Reply:
We (me and @Piotrk39 ) discussed to a add a test here for around ~24 k people instead of the 100k one. Since this is such a critical part of the platform it is valuable to test something